weblate_web/templates/snippets/pricing.html

```diff
@@ -2,10 +2,10 @@
 {% load humanize %}
 	<section class="pricing">
     	<div class="row">
-            <h2 class="section-title">{% trans "<strong>Gratis</strong> or <strong>Premium</strong> — it’s up to you" %}</h2>
+            <h2 class="section-title">{% trans "<strong>Gratis</strong> or <strong>Premium</strong>—it’s up to you" %}</h2>
```

Comment:
I don't think selling price focuses on the service offered. (This is not the Wikipedia model)
How about "for your every need" or similar?